docker-compose.yml

```diff
@@ -24,11 +24,7 @@ services:
       context: ./ui/indexer
     depends_on:
       - "elasticsearch"
-    command: ["/utils/wait-for-it.sh", "elasticsearch:9200",
-      "--timeout=0",
-      "--", "/utils/index.py",
-      "data/vocabulary_ids.txt",
-      "data/vocabulary_bow.txt",
-      "elasticsearch"]
+    command: ["sh","-c",
+"/utils/wait-for-it.sh elasticsearch:9200 --timeout=0 -- /utils/index.py data/vocabulary_ids.txt data/vocabulary_bow.txt elasticsearch && echo Now browse to http://localhost:8080/ for the user interface."]
```

Comment:
I prefer to break this up into an array with its elements on different lines, it's easier to diff that way.

Reply:
Never mind, I just noticed that dowsnt work, apparently